coreblocks/lsu/dummyLsu.py

```
@@ -283,3 +286,13 @@ def _(arg):
             m.d.sync += reserved.eq(0)
 
         return m
+
+
+class LSUBlockComponent(BlockComponentParams):
+    def get_module(self, gen_params: GenParams, dependencies: ComponentDependencies) -> FuncBlock:
+        if dependencies.wishbone_bus is None:
+            raise Exception("LSU requires wishbone bus, but it was not provided as dependency")
```

There should be a way to avoid writing this. Maybe raise the exception from `ComponentDependencies`?